docs/tools/vdb_table/data/milvus.json

```diff
@@ -133,21 +133,21 @@
     "comment": ""
   },
   "sharding": {
-    "support": "",
-    "source_url": "",
+    "support": "full",
+    "source_url": "https://milvus.io/docs/glossary.md#Sharding",
     "comment": ""
   },
   "doc_size": {
-    "bytes": 0,
-    "unlimited": false,
+    "bytes": "Unlimited",
+    "unlimited": true,
     "source_url": "",
     "comment": ""
   },
   "vector_dims": {
-    "value": 32768,
+    "value": "Unlimited",
     "unlimited": false,
     "source_url": "",
```

Comment:
any link to docs for proxy / config mentioned in the comment?

Reply:
see comments

Reply:
I saw the comment, what I'm asking for is a link to put into source_url.

I did a bit of searching, I think this is the right link:
https://milvus.io/docs/latest/configure_proxy.md#proxymaxDimension

Reply:
Oh and you might want to update: https://milvus.io/docs/product_faq.md#What-is-the-maximum-vector-dimension-supported-in-Milvus

Reply:
OK, will update soon